src/reason_config.ml

```diff
@@ -3,7 +3,9 @@
  *)
 
 let recoverable = ref false
+let add_printers = ref false
```

Comment:
In the long run, once the feature is stable, we should probably turn on the default behavior here. 

We will need create an option in build systems to turn on this feature (so that people using rebuild can have a way to try this out). Let's file an issue once this is merged.